packages/react/src/contents/hooks/__tests__/useSeofiles.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { cleanup, renderHook } from '@testing-library/react';
+import { fetchSEOFiles } from '@farfetch/blackout-redux';
+import { mockInitialState } from './__fixtures__/useSeoMetadata.fixtures.js';
+import { seoFilesQuery as query } from 'tests/__fixtures__/contents/seoFiles.fixtures.mjs';
+import { useSeoFiles } from '../index.js';
+import { withStore } from '../../../../tests/helpers/index.js';
+
+jest.mock('@farfetch/blackout-redux', () => {
+  const original = jest.requireActual('@farfetch/blackout-redux');
+
+  return {
+    ...original,
+    fetchSEOFiles: jest.fn(() => ({ type: 'foo-bar' })),
+  };
+});
+
+describe('useSeoFiles', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => cleanup());
+
+  it('should return null if there are no results', () => {
```

Comment:
By default, if no request was made, it should return undefined rather than null. This distinction helps the caller differentiate between the cases where a request was made but yielded no results and the case where no request was made at all.

Reply:
Resolved! ✅